From 00a0d5d7be8a5ed37d827644c2abf4efa2f6ea11 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 18:58:34 -0400
Subject: [fs] btrfs: fix unused variables in qgroup.c

Message-id: <1407179109-13714-28-git-send-email-sandeen@redhat.com>
Patchwork-id: 86979
O-Subject: [RHEL7.1 PATCH 027/422] btrfs: fix unused variables in qgroup.c
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Valentina Giusti <valentina.giusti@microon.de>

commit a3df41ee377b2766a7525f9ca05207698efe4551
Author: Valentina Giusti <valentina.giusti@microon.de>
Date:   Mon Nov 4 22:34:29 2013 +0100

    btrfs: fix unused variables in qgroup.c

    Use otherwise unused local variables slot in update_qgroup_limit_item and
    in update_qgroup_info_item, and remove unused variable ins from
    btrfs_qgroup_account_ref.

    Signed-off-by: Valentina Giusti <valentina.giusti@microon.de>
    Signed-off-by: Josef Bacik <jbacik@fusionio.com>
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/qgroup.c b/fs/btrfs/qgroup.c
index 4e6ef49..bd0b058 100644
--- a/fs/btrfs/qgroup.c
+++ b/fs/btrfs/qgroup.c
@@ -644,8 +644,7 @@ static int update_qgroup_limit_item(struct btrfs_trans_handle *trans,
 
  l = path->nodes[0];
  slot = path->slots[0];
- qgroup_limit = btrfs_item_ptr(l, path->slots[0],
-          struct btrfs_qgroup_limit_item);
+ qgroup_limit = btrfs_item_ptr(l, slot, struct btrfs_qgroup_limit_item);
  btrfs_set_qgroup_limit_flags(l, qgroup_limit, flags);
  btrfs_set_qgroup_limit_max_rfer(l, qgroup_limit, max_rfer);
  btrfs_set_qgroup_limit_max_excl(l, qgroup_limit, max_excl);
@@ -687,8 +686,7 @@ static int update_qgroup_info_item(struct btrfs_trans_handle *trans,
 
  l = path->nodes[0];
  slot = path->slots[0];
- qgroup_info = btrfs_item_ptr(l, path->slots[0],
-     struct btrfs_qgroup_info_item);
+ qgroup_info = btrfs_item_ptr(l, slot, struct btrfs_qgroup_info_item);
  btrfs_set_qgroup_info_generation(l, qgroup_info, trans->transid);
  btrfs_set_qgroup_info_rfer(l, qgroup_info, qgroup->rfer);
  btrfs_set_qgroup_info_rfer_cmpr(l, qgroup_info, qgroup->rfer_cmpr);
@@ -1349,7 +1347,6 @@ int btrfs_qgroup_account_ref(struct btrfs_trans_handle *trans,
         struct btrfs_delayed_ref_node *node,
         struct btrfs_delayed_extent_op *extent_op)
 {
- struct btrfs_key ins;
  struct btrfs_root *quota_root;
  u64 ref_root;
  struct btrfs_qgroup *qgroup;
@@ -1363,10 +1360,6 @@ int btrfs_qgroup_account_ref(struct btrfs_trans_handle *trans,
 
  BUG_ON(!fs_info->quota_root);
 
- ins.objectid = node->bytenr;
- ins.offset = node->num_bytes;
- ins.type = BTRFS_EXTENT_ITEM_KEY;
-
  if (node->type == BTRFS_TREE_BLOCK_REF_KEY ||
      node->type == BTRFS_SHARED_BLOCK_REF_KEY) {
   struct btrfs_delayed_tree_ref *ref;
-- 
1.7.1